nevergrad/functions/ac/ac.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Optimization of the FAO crop management model.
+Based on 
+https://colab.research.google.com/github/thomasdkelly/aquacrop/blob/master/tutorials/AquaCrop_OSPy_Notebook_3.ipynb#scrollTo=YDm931IGNxCb
+"""
+
+from nevergrad.parametrization import parameter
+from ..base import ExperimentFunction
+
+# pylint: disable=too-many-locals,too-many-statements
+
+# Inspired by
+# https://colab.research.google.com/github/thomasdkelly/aquacrop/blob/master/tutorials/AquaCrop_OSPy_Notebook_3.ipynb#scrollTo=YDm931IGNxCb
+
+from aquacrop.classes import *
+from aquacrop.core import *
```

Comment:
please avoid star imports, it makes it less clear what comes from what.

Reply:
Ok. However I am getting farther from the original code, which is a bit sad. I keep the original code in comment.